randovania/games/dread/json_data/Dairon.txt

```diff
@@ -278,7 +278,7 @@ Extra - asset_id: collision_camera_003
   * Extra - actor_def: actordef:actors/props/dooremmy/charclasses/dooremmy.bmsad
   > Elevator to Ferenia - East Transport to Darion
       After Dairon - Pull Grapple Block below Ferenia Elevator
-  > Event - Pull Grapple Block
+  > Event - Pull Grapple Block below Elevator to Ferenia
```

Comment:
The event nodes don't have to match the resource name precisely. (if that were the case, there'd be a rule like we do for docks).
_Resources_ have to be unique and thus include the room name, but a shorter node name reads better.

Whenever Randovania displays a node, it takes care to include the area (and even world) names.